meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-kernel/linux/linux-lmp-ti-staging_6.6.bb

```diff
@@ -1,13 +1,13 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
-include recipes-kernel/linux/kmeta-linux-lmp-6.1.y.inc
+include recipes-kernel/linux/kmeta-linux-lmp-${PV}.y.inc
 
 KERNEL_REPO ?= "git://git.ti.com/git/ti-linux-kernel/ti-linux-kernel.git"
 KERNEL_REPO_PROTOCOL ?= "https"
-KERNEL_BRANCH ?= "ti-linux-6.1.y"
+KERNEL_BRANCH ?= "ti-linux-6.6.y"
 
-LINUX_VERSION ?= "6.1.83"
-SRCREV_machine ?= "c1c2f1971fbf6ddad93a8c94314fe8221e7aa6be"
+LINUX_VERSION ?= "6.6.44"
+SRCREV_machine ?= "e3b431a6194fc1b074802d20c9eede8442382b2a"
```

Comment:
Please add the reason for the change as part of the commit message.

Reply:
I add some few additional info, but I don't know if there is a reason to go from 6.1 to 6.6 (other than only bumping)